src/CLI/Command/Check.php

```diff
@@ -95,6 +99,12 @@ protected function configure(): void
                 InputOption::VALUE_OPTIONAL,
                 'Output format: text (default), json, gitlab',
                 'text'
+            )
+            ->addOption(
+                self::AUTOLOAD_PARAM,
+                'a',
+                InputOption::VALUE_REQUIRED,
```

Comment:
Why this param is required and not optional?

Reply:
That's because if you specify --autoload you need to provide a value for it